app/models/teacher.rb

```diff
@@ -2,12 +2,14 @@
 
 class Teacher < ApplicationRecord
   validates :first_name, :last_name, :email, :status, presence: true
-  validates_inclusion_of :validated, :in => [true, false]
+  validates_inclusion_of :application_status, :in => %w(Validated Denied Pending)
```

Comment:
We should really make this application_status an enum. (We'll talk about this for the next iteration)

Reply:
```rb

enum application_status: [
:validated,
:denied,
:pending
]
```

Later we can use:
`@teacher.validated?` or `@teacher.validated!`